src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "2.0.0b2"
+VERSION = "2.0.0b3"
```

Comment:
Since 2.0.0b3 has been existed, please update it to a new version.

Reply:
Updated